### PR TITLE
Field Terminology in QPACK Security Considerations

### PR DESCRIPTION
**Pull request by MikeBishop**
Noticed that the Security Considerations seems to have come in after and without the "field sections" terminology, so I did a pass through there.  This will probably have a few conflicts with #4007.

### CODE REVIEW

File: draft-ietf-quic-qpack.md [hunk 1]
```diff
@@ -1225,12 +1225,12 @@ a linear-time attack.
 ## Applicability to QPACK and HTTP
 
 QPACK mitigates but does not completely prevent attacks modeled on CRIME [CRIME]
-by forcing a guess to match an entire header field value, rather than individual
+by forcing a guess to match an entire field value, rather than individual
```
Comment (afrind) [inline draft-ietf-quic-qpack.md:1228]: i think it has to match a field line, not field value (here and below)?  And technically the original HPACK text should have said entire header (you have to match the name and value).

File: draft-ietf-quic-qpack.md [hunk 2]
```diff
@@ -1253,51 +1253,51 @@ different web origins {{?RFC6454}} are made by mutually distrustful entities.
...
 tagged as being public. For example, a web browser might make the values of the
 Accept-Encoding header field available in all requests.
 
-An encoder without good knowledge of the provenance of header fields might
-instead introduce a penalty for a header field with many different values, such
-that a large number of attempts to guess a header field value results in the
-header field not being compared to the dynamic table entries in future messages,
-effectively preventing further guesses.
+An encoder without good knowledge of the provenance of field values might
+instead introduce a penalty for a field name with many different values, such
```
Comment (LPardue) [inline draft-ietf-quic-qpack.md:1275]: is this correct, or a typo?

Comment (MikeBishop) [reply draft-ietf-quic-qpack.md:1275]: Shorthand, I think.  What it's trying to express is that if there are many field lines in the dynamic table which have the same field name and distinct values, you might with some probability ignore the dynamic table when encoding.  The more entries, the higher the probability you should ignore them all to frustrate an attacker.

**Comment by MikeBishop** [comment]
Sorry, this was a mistake; I think I've backed it out now.

**Pull request merged**